### PR TITLE
Import division from future

### PR DESCRIPTION
**Pull request by AntoineDao**
I noticed some bugs in Dragonfly because I was testing with integer values and the divisions were not running as expected eg: [this stack overflow post](https://stackoverflow.com/questions/183853/in-python-2-what-is-the-difference-between-and-when-used-for-division).

I propose we always import divsion from __future__ and specifically use `\\` when we want to use integer division. Let me know what you think.

```python
from __future__ import division
```

### CODE REVIEW

**Comment by saeranv** [comment]
@AntoineDao 

What exactly is the bug that is occurring in Dragonfly? Is it because integer values are being used as inputs (which shouldn't cause an integer division error) or testing in Python 3 or IronPython is causing integer division to fail?

If the integer division isn't returning a floored integer in IronPython or Python 3, then yes, we'll have to change the code. 

Either importing the future module, or maybe something like this: `int(math.floor(a/b))` which might save us from importing a new library. 

-S

**Comment by AntoineDao** [comment]
The issue was that I was using integers as input but that I was expecting them to behave like float divisions (eg: 5/2 = 2.5 rather than 5/2=2, or worse yet smaller/bigger = 0). The reason this happened is because I used Dragonfly without Rhino and thus entered most of my values manually. I reckon this is a valid use case as it would be cool to have dragonfly/uwg run as a web service for example. 

I also reckon this will help transition to python 3 and ensure similar behaviour.

**Comment by saeranv** [comment]
@AntoineDao 

Oh I see, yes that would do it. I would break this up into two issues:
1. Types of user inputs
2. Integer division

I consider types of user inputs as a slightly different issue then integer division, because there are other properties of integers vs floats (i.e equalities) that can cause downstream problems, not considered by integer division.  

I started an issue to address this separately here: https://github.com/ladybug-tools/uwg/issues/73

For the second issue;

If all our types are correctly cast into floats, integers, then using the `//` method should be fine, as long as we identify all the instances of integer division (kind of impossible to do since some variables/properties are cast as ints early on before they are divided). 

I dislike that importing the future module changes the existing behavior of  `/’ but if we move to Python 3 that’s the hand we’re dealt! In this case, I suppose we will have to have faith in our test reports, and make the change and hope no subtle error is introduced in the calculations where we don’t catch integer division. 

In summary, all looks good so I'll merge this. 

Thanks, 

-S

**Comment by AntoineDao** [comment]
Yep that all makes sense. It is annoying that behaviour is different between the two pythons but overall I reckon this project will benefit enormously from prepping for python 3 early as python 2 will no longer be supported next year... Now for strict type casting in Python :p
